geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -30,6 +33,7 @@
 import redis.clients.jedis.JedisCluster;
 import redis.clients.jedis.Protocol;
 
+import org.apache.geode.redis.ConcurrentLoopingThreads;
 import org.apache.geode.redis.RedisIntegrationTest;
 import org.apache.geode.test.awaitility.GeodeAwaitility;
 
```

Comment:
Not code you added, but we can remove the `REDIS_CLIENT_TIMEOUT` constant below from this class as it's defined in `RedisClusterStartupRule`. We can also replace the "localhost" String in the `HostAndPort` constructor with the `BIND_ADDRESS` constant in `RedisClusterStartupRule`.